index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Daniella Skinner's Portfolio</title>
+  <meta name="viewport" content="width=device-width">
+  <link rel="stylesheet" type="text/css" href="normalize.css">
+  <link rel="stylesheet" type="text/css" href="portfolioStyles.css">
+</head>
+
+<body>
+	<nav>
+		<h2>Daniella Skinner
+		<p>Full stack web developer in training</p>
```

Comment:
Shouldnt have a p tag inside an h2 as they have conflicting default styles, best practice is just to use one